.gitignore

```diff
@@ -1,7 +1,7 @@
 target/
 javy.wat
 javy.wasm
-index.js
+/index.js
```

Comment:
This is so we only ignore a `index.js` file in the repo root. I opted to use `index.js` as the filename in the benchmarking suite. I can use a different name if anyone has strong opinions.